mandatory/2-function-creation.js

```diff
@@ -25,8 +37,15 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
-
+function tidyUpStrings(arrayOfStrings) {
+  let array = arrayOfStrings;
```

Comment:
arrays are non-primitive data types, they are passed by reference not by value. ```array``` is not a copy of ```arrayOfStrings```, ```array``` is a second reference to ```arrayOfStrings```.